tei_adapter/tei_id_extractor/src/main/resources/application.conf

```diff
@@ -1,3 +1,14 @@
 aws.sqs.queue.url=${?queue_url}
 aws.sns.topic.arn=${?topic_arn}
 tei.github.token=${?github_token}
+aws.rds.host=${?db_host}
+aws.rds.password=${?db_password}
+aws.rds.port=${?db_port}
+aws.rds.username=${?db_username}
+aws.rds.maxConnections=${?max_connections}
+tei.id_extractor.bucket=${?bucket}
+tei.id_extractor.database=pathid
```

Comment:
Why not also parameterize these?

Reply:
no reason really. They're not parametrised in the id minter and I just copied that I think. Will change